data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	(Tell her about the Coalition.)`
+				`	"No actually, I'm afraid I don't know anyone who could help you."`
+					decline
+			label coalition
+			`	You begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them. At first Anna does not believe you; but you reassure her it's real. You also mention the population of the Coalition and the novelty of an "alien" author coming into the area.`
+			`	"Will they really like my books?"`
+			choice
+				`	"They will, hopefully."`
+				`	"Well, it's better to try than to just give up."`
+			`	"These stories might just inspire my next novel. But, they wouldn't speak our language. How are they going to read my books?" You reply, "They have a translation device."`
+			`	"That's reassuring... but, how am I even going to get there?" she queries.`
+			choice
+				`	"I'll take you there."`
+				`	"I don't think you can."`
+					goto decline
+			`	"Do I have to pay anything? As a struggling author, I can't afford off-world transport at all, let alone to a completely different segment of the galaxy! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	As you wait for Anna, you look over your maps of Coalition space to double-check what planet the Coalition book club is on. After a quick look, you find <destination>.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open it to let her inside, lead her to her bunk, and inform her of the destination. You prepare to leave the spaceport.`
+				accept
+			label decline
+			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. Thank you for the stories, at least." She leaves.`
+				decline
+	on enter
+		system
+			government "Coalition" "Heliarch"
+		conversation
+			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
+			choice
+				`	"My jump drive would like to welcome you to Coalition space."`
+				`	"That was a jump drive."`
+				`	"That's what happens every time I go to Coalition space with a jump drive."`
+			`	"Wow. I can't believe this is happening! I'm so excited!" Anna looks out the window. "What are those golden ships?"`
+			`	"They're Heliarch vessels," you reply.`
```

Comment:
I'm not sure these lines do us much good. Coalition ships have very little color, so it would be awkward for Anna to call Heliarch ships "golden" when they have so little swizzled surface area; the player could be very confused as they might not ever have even noticed that Coalition/Heliarch ships swizzle *at all*.

I think it would flow better to just continue on with her amazement and curiosity about the JD.